UnitTests/BraintreeShopperInsightsTests/BTShopperInsightsClient_Tests.swift

```diff
@@ -221,8 +221,8 @@ class BTShopperInsightsClient_Tests: XCTestCase {
     }
     
     func testSendPayPalSelectedEvent_sendsAnalytic() {
-        sut.sendPayPalSelectedEvent()
-        XCTAssertEqual(mockAPIClient.postedAnalyticsEvents.first, "shopper-insights:paypal-selected")
+        sut.sendSelectedEvent(for: .payPal)
```

Comment:
are we also able to assert on the button type being set correctly here and below in the venmo test?

Reply:
done!